docs/contributing/ui-testing.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Ui Testing
+---
+## What is UI testing?
+Manual tests are time-consuming and sometimes error occurs. A more efficient approach is to create UI tests so that user tasks are executed in an automated test. With UI test, tests can be executed quickly and reliably in a repeatable way. Simply stated, first priority considering is two things.
```

Comment:
Enact supports ui-test-utils package which includes the common WebDriver configurations and some utility modules for executing automated UI tests. With UI test, you can execute user interaction testing quickly and reliably.